sigs/proposed.md

```diff
@@ -0,0 +1,14 @@
+## Initial SIGS
```

Comment:
Why is this removed from the main doc, and not linked from anywhere?
This is the list of SIGs that was approved by the TOC.  I think it should:

1. remain the canonical list of SIGs, clearly indicated as such, 
2. be updated to indicate the TOC liaisons that have been appointed to each by the TOC.
3. be updated to link to the details of each SIG as those get formulated (or indicate that the SIG is still nascent).
4. be updated if and when SIG's are renamed, split, merged etc.

I would suggest splitting out the formatting-only changes into a separate PR, and then we can do the other restructuring/updates separately.